src/Compilers/VisualBasic/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.vb

```diff
@@ -97,13 +97,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
             End If
 
+            Dim vbProperty = TryCast(symbol, PropertySymbol)
+            Dim countOfCustomModifiersPrecedingByRef As UShort = If(vbProperty Is Nothing, 0US, vbProperty.CountOfCustomModifiersPrecedingByRef)
```

Comment:
We have just decided to expose this information through the public API. I will remove this code in the process in a separate change.